tests/stave_tests.ts

```diff
@@ -478,7 +543,7 @@ function drawVolta(options: TestOptions, contextBuilder: ContextBuilder): void {
   // bar 8 - juxtapose eighth measure
   const mm8 = new Stave(mm7.getWidth() + mm7.getX(), mm1.getY(), 60);
   mm8.setEndBarType(BarlineType.DOUBLE);
-  mm8.setRepetitionTypeRight(Repetition.type.DS_AL_CODA, 25);
+  mm8.setRepetitionType(Repetition.type.DS_AL_CODA, 0);
```

Comment:
Since the second parameter is always `0`, can we make it a default. Also, I think we should have a couple of tests for non-zero parameters.

Reply:
Done